### PR TITLE
Add style bindings

### PR DESCRIPTION
**Pull request by irustm**

### CODE REVIEW

File: snippets/style-bibdings.md [hunk 1]
```diff
@@ -0,0 +1,65 @@
...
+
+myStyles = {
+'background-color': 'black',
+'color': 'white',
+'font-size': '25px',
+}
+
+<!--Svg attribute bindings-->
+
+Before:
```
Comment (kirjs) [inline snippets/style-bibdings.md]: This is cool should be a separate tip

Comment (irustm) [reply snippets/style-bibdings.md]: remove svg attribute bindings

File: snippets/style-bibdings.md [hunk 2]
```diff
@@ -0,0 +1,49 @@
+---
+title: Style bindings
+tags:
+  - tips
```
Comment (kirjs) [inline snippets/style-bibdings.md]: Update tags

File: snippets/style-bibdings.md [hunk 3]
```diff
@@ -0,0 +1,49 @@
...
+
+<p [style.background-color]="'green'">
+  I am in green background
+</p>
+
+<p [style.font-size.px]="isImportant ? '30' : '16'">
+  May be important text.
+</p>
+
+<p [ngStyle]="myStyles">
```
Comment (kirjs) [inline snippets/style-bibdings.md]: Maybe move to a separate template

**Review by kirjs — APPROVED** [review]

**Pull request merged**